net/libfab/libfab.c

```diff
@@ -438,15 +442,15 @@ static void libfab_tm_event_post(struct m0_fab__tm *tm,
  */
 static void libfab_tm_buf_timeout(struct m0_fab__tm *ftm)
 {
-	struct m0_net_transfer_mc *net = ftm->ftm_net_ma;
+	struct m0_net_transfer_mc *net = ftm->ftm_ntm;
 	int                        i;
 	m0_time_t                  now = m0_time_now();
 
 	M0_PRE(libfab_tm_invariant(ftm));
 	for (i = 0; i < ARRAY_SIZE(net->ntm_q); ++i) {
 		struct m0_net_buffer *nb;
 
-		m0_tl_for(m0_net_tm, &ftm->ftm_net_ma->ntm_q[i], nb) {
+		m0_tl_for(m0_net_tm, &ftm->ftm_ntm->ntm_q[i], nb) {
```

Comment:
s/&ftm->ftm_ntm->ntm_q[i]/&net->ntm_q[i]/

Reply:
To iterate a list, some lock must be held.
We must take a lock here, or assert this lock is already held.

Reply:
Done.